advanced/Scripts/piholeDebug.sh

```diff
@@ -151,6 +160,17 @@ function hostnameCheck {
     fi
 }
 
+function portCheck {
+    echo "############################################################" >> ${DEBUG_LOG}
+	echo "########           Open Port Information           #########" >> ${DEBUG_LOG}
+	echo "############################################################" >> ${DEBUG_LOG}
+
+    echo "::: Writing local server ports to logfile"
+
+    ${SUDO} netstat -tulpn >> ${DEBUG_LOG}
```

Comment:
We're looking for the owners of the processes at port 80 and 53 to see if they are who we expect, or if they are owned by another daemon. I guess I could change it around to run pidof or some other attempt with systemctl, but just getting a response back isn't what were looking for in this section.

There's a nslookup series of tests and I'll be adding in a couple of dig commands specific to dnsmasq to pull some metrics out of the process